src/python/pants/reporting/zipkin_reporter.py

```diff
@@ -0,0 +1,87 @@
+# coding=utf-8
+# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import requests
+from py_zipkin import Encoding
+from py_zipkin.transport import BaseTransportHandler
+from py_zipkin.zipkin import zipkin_span
+
+from pants.reporting.reporter import Reporter
+
+
+class HTTPTransportHandler(BaseTransportHandler):
+  def __init__(self, endpoint, encoding):
+    self.endpoint = endpoint
+    self.encoding = encoding
+
+  def get_max_payload_bytes(self):
+    return None
+
+  def send(self, payload):
+    try:
+      requests.post(
+        # Twitter endpoint 'https://zipkin.twitter.biz/api/v1/spans',
+        # Local endpoint 'http://localhost:9411/api/v1/spans',
```

Comment:
Either remove this, or replace it with a comment that explains how to start a local zipkin